### DISCUSSION TITLE
Change editor palette bank

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2379] [Ideas]
Is it possible to change the editor theme in config cart without changing the bank0 palette?

**Comment by Skeptim** [comment]
Note that changing the bank0 palette in the config cart won't change the bank0 palette in any other cart.

Are you asking because you find it too limiting ?

It's currently only possible for what concerns the text editor with the parameters listed here: https://github.com/nesbox/TIC-80/wiki/System-Configuration-Cartridge#color-parameters and a few sprites of the config cart like the gamepad buttons, as you probably already know. But that's all for now.
However, some improvements are planned https://github.com/nesbox/TIC-80/issues/2341 and you could add your suggestions in this issue.
